config/environments/production.rb

```diff
@@ -85,7 +85,15 @@
       rescue
         $stderr.puts "Error running migration! Please review database configuration"
       end
+      if User.count == 0
```

Comment:
Nope. Only seed users with `rake db:seed`. That's really unexpected from a Rails app.